### PR TITLE
Reorganize code

### PR DESCRIPTION
**Pull request by viruschidai**

### CODE REVIEW

File: src/styles/Icons.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { Platform } from 'react-native';
+
+export const weatherIconsFontFamily = (Platform.OS === 'android') ?  'weathericons' : 'Weather Icons';
```
Comment (JakeLin) [inline src/styles/Icons.js:3]: Is possible to use `Icons.ios.js` and `Icons.android.js`?

**Comment by JakeLin** [comment]
👍

File: src/styles/Icons.js (hunk 1 above)
Comment (viruschidai) [inline src/styles/Icons.js:3]: did not try, it should work according to the doc. This icon is too  small and I don't really wanna create two files for it ATM

Comment (JakeLin) [inline src/styles/Icons.js:3]: I am OK with that atm

**Pull request merged**